src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
+    "modification-queue-size": "2048"
+  }
+}`;
+
+export const remoteStore = `"remote-store": {
+  "cache": "mycache",
+  "raw-values": "true",
+  "remote-server": [
+    {
+      "host": "one",
```

Comment:
I know these values are from the docs, but we should use an ip address for the hosts. I would also just include a single server, as I think this will fit most use-cases. The definition being inside an array implies that more than one object can be defined.

Reply:
Should the cache be like this? with a single host

```
"remote-store": {
  "cache": "mycache",
  "remote-server": [
    {
      "host": "one",
      "port": "12111"
    },
  ]
}
```

Reply:
You just need to remove the last comma:

```json
"remote-store": {
  "cache": "mycache",
  "remote-server": [
    {
      "host": "one",
      "port": "12111"
    }
  ]
}
```

Reply:
> I know these values are from the docs, but we should use an ip address for the hosts.

I should have updated my example to include ^. Also, the default port for Hot Rod is `11222` so we should utilise that in the template:

```json
"remote-store": {
  "cache": "mycache",
  "remote-server": [
    {
      "host": "127.0.0.1",
      "port": "11222"
    }
  ]
}
```